src/Middleware/Spa/SpaProxy/src/SpaProxyLaunchManager.cs

```diff
@@ -102,6 +90,19 @@ public async Task<bool> IsSpaProxyRunning(CancellationToken cancellationToken)
             }
         }
 
+        private static HttpClient CreateHttpClient()
+        {
+            var httpClient = new HttpClient(new HttpClientHandler()
+            {
+                // It's ok for us to do this here since this service is only plugged in during development.
+                ServerCertificateCustomValidationCallback = HttpClientHandler.DangerousAcceptAnyServerCertificateValidator
+            });
+            // Certain frontend build tools rely on the the ACCEPT Header being set, otherwise these tools
+            // might be unable to perform their client-side fallback logic.
+            httpClient.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("*/*"));
```

Comment:
Given that we are changing this, can we give this a quality value lower than 1.0? Just in case it affects other build servers. Something like `0.1-0.5` should do it.

```suggestion
            httpClient.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("*/*", 0.1));
```